src/olympia/api/serializers.py

```diff
@@ -60,7 +60,14 @@ def fake_object(self, data):
     def handle_date(self, value):
         if not value:
             return None
-        return datetime.strptime(value, u'%Y-%m-%dT%H:%M:%S')
+
+        # Don't be picky about microseconds here. But we get them some time
+        # so we have to support them. Maybe there's a nicer way of doing
+        # this? (cgrebs)
```

Comment:
Stripping them now and added a test in 989cdd5. I figured it'd be best to do that here than to strip that everytime on the input-side.